### PR TITLE
Update BT shell for LE Audio

### PR DESCRIPTION
**Pull request by Thalley**
Moves all LE audio related configs to a new .conf file and update the naming in testcase.yaml.

### CODE REVIEW

File: tests/bluetooth/shell/audio.conf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+CONFIG_BT=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_FIXED_PASSKEY=y
+CONFIG_BT_ATT_PREPARE_COUNT=2
+CONFIG_BT_SHELL=y
+CONFIG_BT_DEVICE_NAME="test shell"
```
Comment (asbjornsabo) [inline tests/bluetooth/shell/audio.conf:15]: Do we want "audio" in the shell name for this config?

**Review by asbjornsabo — APPROVED** [review]
One question, otherwise LGTM

File: tests/bluetooth/shell/audio.conf (hunk 1 above)
Comment (Thalley) [reply tests/bluetooth/shell/audio.conf]: I think that's a good idea; added.

**Comment by stephanosio** [comment]
@nashif strange ... another stuck buildkite build here too.

**Comment by Thalley** [comment]
> @nashif strange ... another stuck buildkite build here too.

@stephanosio What's the issue? Haven't seen this before.

**Review by alwa-nordic — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**